### ISSUE TITLE
Dialogs library bring to front doesn't work in Windows Server 2016

### ISSUE THREAD

**Issue by jpeltonen** [robotframework/robotframework#3492]
Dialog was left minimized after calling "Execute Manual Step". 

Could fix by modifying C:/Python368/Lib/site-packages/robot/libraries/dialogs_py.py (commented lines are original which didn't work)

Original code:
```
    def _bring_to_front(self):
        self.attributes('-topmost', True)
        self.attributes('-topmost', False)
```
Fix that seem to work:

```
    def _bring_to_front(self):
        self.lift()
        self.attributes('-topmost', True)
        self.after_idle(self.attributes, '-topmost', False)
```
Pull request to fix this #3496

**Comment by pekkaklarck** [comment]
Do you mean this can be fixed by commenting code? If not, please provide a diff showing the actual changes or, even better, a pull request.

**Comment by jpeltonen** [comment]
No this is just issue. Not pull request yet. Don't know if it depends on operating system also and how it works with Linux.

**Comment by pekkaklarck** [comment]
This was fixed by PR #3496. Thanks @jpeltonen!
